x/pairing/keeper/pairing_subscription_test.go

```diff
@@ -445,7 +445,7 @@ func TestPairingNotChangingDueToCuOveruse(t *testing.T) {
 	client1Acct, client1Addr := ts.GetAccount(common.CONSUMER, 0)
 
 	// add 10 months to the subscription
```

Comment:
coment and call mismtach

Reply:
done. see https://github.com/lavanet/lava/commit/3b37e3ad41d3e548c74b458e0a85170abd0134b0